packages/stark-ui/src/modules/minimap/assets/translations/en.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @ignore
+ */
+export const translationsEn: object = {
+	STARK: {
+		TABLE: {
+			TOGGLE_COLUMNS: "Column filters"
```

Comment:
Could you please move this translation in stark-core/src/common/translations/translations ? 😊 
Like this all the translations are in the same location